en/02_Developer_Guides/05_Extending/05_Injector.md

```diff
@@ -334,17 +407,24 @@ Calling..
 use App\MyController;
 use SilverStripe\Core\Injector\Injector;
 
-// sets up MyController as a singleton
 $controller = Injector::inst()->get(MyController::class);
 ```
 
-Would setup the following
-
-* Create an object of type `App\MyController`
-* Look through the **dependencies** and call `get('App\PermissionService')`
-* Load the configuration for PermissionService, and create an object of type `App\RestrictivePermissionService`
-* Look at the properties to be injected and look for the config for `App\MySQLDatabase`
-* Create a `App\MySQLDatabase` class, passing `dbusername` and `dbpassword` as the parameters to the constructor.
+Would perform the following steps:
+
+* Fetches a singleton of the `App\MyController` service
+* Assuming this is the first time a `App\MyController` singleton is being instantiated:
```

Comment:
I'll change this a bit to 
> If there no existing instance for the `App\MyController` singleton

This is because the singleton doesn't have to be an instance of `App\MyController`